features/step_definitions/common.rb

```diff
@@ -130,6 +130,14 @@
   cb[clipboard_name] = eval(what)
 end
 
+Given /^(I|admin) save custom resource `(.+?)` to file>(.+)$/ do |what,filepath|
```

Comment:
A new regexp `I|admin` introduced, but not deal with. Thus 3 regexp but only 2 params, which will cause error. 
I think we can simply use `I` here to make it simple.

Reply:
Also, there is a small typo in the commit message `Add a step for saving CR to a file ^Crectly`

Reply:
oops, thanks, i will use just `I`